--- a/src/other/Shake.cpp
+++ b/src/other/Shake.cpp
@@ -21,11 +21,11 @@
     // add custom bezier (and read it after config reload)
     static auto bezier = "dynamic-cursors-magnification";
-    g_pAnimationManager->addBezierWithName(bezier, {0.22, 1.0}, {0.36, 1.0});
-    static const auto LISTENER = Event::bus()->m_events.config.reloaded.listen([&]() -> void { g_pAnimationManager->addBezierWithName(bezier, {0.22, 1.0}, {0.36, 1.0}); });
+    g_pAnimationManager->addBezierWithName(bezier, {0.0, 0.0}, {1.0, 1.0});
+    static const auto LISTENER = Event::bus()->m_events.config.reloaded.listen([&]() -> void { g_pAnimationManager->addBezierWithName(bezier, {0.0, 0.0}, {1.0, 1.0}); });
 
     // wtf is this struct, what is pValues?
     static SP<SAnimationPropertyConfig> properties = makeShared<SAnimationPropertyConfig>();
     properties->internalBezier                     = bezier;
-    properties->internalSpeed                      = time / 100.f;
+    properties->internalSpeed                      = 400.0;
     properties->internalEnabled                    = 1;
     properties->pValues                            = properties;
